native/src/main/scala/sqs4s/api/errors.scala

```diff
@@ -0,0 +1,49 @@
+package sqs4s.api
+
+import cats.Show
+import cats.implicits._
+import org.http4s.Status
+
+import scala.xml.Elem
+
+object errors {
+  sealed trait SqsError extends Exception
+
+  case class BasicAwsSqsError(status: Status, raw: Elem) extends SqsError {
+    override def getMessage: String =
+      s"""Http code: ${status.code}
+         |Raw: ${raw.toString}
+     """.stripMargin
+  }
+
+  case class AwsSqsError(
+    status: Status,
+    `type`: String,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [`type` is a reserved keyword don't use it as a name](https://app.codacy.com/manual/d2a4u/sqs4s/pullRequest?prid=4059007)